### PR TITLE
fix(ie11): preact-compat algolia fork with #423

### PR DESCRIPTION
**Pull request by bobylito**
This is to fix an [issue of preact-compat](https://github.com/developit/preact-compat/pull/423) on IE11. It uses an algolia fork of preact-compat, that we need to replace once the new version is out (3.17.1 or 3.18.0)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "instantsearch.js",
-  "version": "2.2.0",
+  "version": "2.2.1-beta.0",
```
Comment (iam4x) [inline package.json]: this should only change with the build script

Comment (bobylito) [reply package.json]: That's alright, since it's gonna be 2.2.1 anyway.

**Comment by iam4x** [comment]
Failling on the commit:

```
Checking commit message for 2c5abfc3e7315d6a723b96a539a09857d48e101d... KO (format): v2.2.1-beta.0
```

**Comment by bobylito** [comment]
Well it seems that it might not fix what is supposed to. Will check that tomorrow.

**Comment by bobylito** [comment]
Actually the library works with IE11. No need for this PR.

**Pull request closed without merging**